tests/Unit/Crypt/HashTest.php

```diff
@@ -95,6 +95,33 @@ public static function hashData()
                 'The quick brown fox jumps over the lazy dog.',
                 '87a7ff096082e3ffeb86db10feb91c5af36c2c71bc426fe310ce662e0338223e217def0eab0b02b80eecf875657802bc5965e48f5c0a05467756f0d3f396faba'
             ),
+            array(
+                'whirlpool',
+                'The quick brown fox jumps over the lazy dog.',
+                '87a7ff096082e3ffeb86db10feb91c5af36c2c71bc426fe310ce662e0338223e217def0eab0b02b80eecf875657802bc5965e48f5c0a05467756f0d3f396faba'
+            ),
+            // from http://csrc.nist.gov/groups/ST/toolkit/documents/Examples/SHA512_224.pdf
```

Comment:
'sha224' has also been added (see https://github.com/phpseclib/phpseclib/pull/956/files#diff-b4b2acaebb5e3d125481ec5da11baff5R1972), but it doesn't have test cases defined here. Is this on purpose?
Please note that its examples are similarly available in http://csrc.nist.gov/groups/ST/toolkit/documents/Examples/SHA224.pdf.